packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -792,14 +792,18 @@ class _CupertinoModalPopupRoute<T> extends PopupRoute<T> {
     this.barrierColor,
     this.barrierLabel,
     this.builder,
+    bool semanticsDismissible,
```

Comment:
Hmm this doesn't work because in the `ModalRoute` abstract class it is defined as simply a getter that returns true, so it then ignores the property that I set in `_CupertinoModalPopupRoute`

Reply:
I believe dart allows overwriting a getter with a final field? We do it elsewhere in the framework and the analyzer seems to be happy about it: https://dartpad.dev/ed41bd0c9b87802c42701569278fcbad

Reply:
Seems like I wasn't able to get this to work, going to leave as-is.